src/host/twitter/utils.ts

```diff
@@ -15,3 +15,17 @@ export const extractTwitterHandleFromPathname = (pathname: string) => {
 export const isTwitterHomePathname = (pathname: string) => {
   return pathname.startsWith('/home');
 };
+
+export const isHandleNode = (element: HTMLElement) => {
+  let currentElement: HTMLElement | null = element;
+  for (let index = 0; index < 10; index++) {
+    currentElement = currentElement.parentElement;
+    if (!currentElement) {
+      return false; // If there are less than 3 parents
```

Comment:
I'm unsure if I understand the comment here, should there be `10` instead of `3`?

Reply:
yeah correct :D

Reply:
instead of fixing comment I moved this magic number to some meaningful constant